plugin/src/main/java/com/denizenscript/denizen/objects/PlayerTag.java

```diff
@@ -1242,9 +1242,18 @@ else if (ban.getExpiration() == null) {
                 return null;
             }
             ListTag list = new ListTag();
-            // TODO: optionally specify world
+            WorldTag world = null;
+            if (attribute.hasContext(1)) {
+                if (WorldTag.matches(attribute.getContext(1))) {
+                    world = (WorldTag) attribute.getContextObject(1);

```

Comment:
As mentioned before: you can't cast objects like that. Use `contextAsType`

Reply:
Copy 👍